### ISSUE TITLE
Update mod for Beat Saber update 0.13

### ISSUE THREAD

**Issue by renssies** [opl-/beatsaber-http-status#19]
It looks like this mod can't be loaded on Beat Saber update 0.13 (the update that introduces the paid DLC)

Running diagnostics in Modsaber installer returns the following:
```Beat Saber
  │ [WARN] Could not load plugin BeatSaberHTTPStatus.Plugin in BeatSaberHTTPStatus.dll! System.TypeLoadException: Could not load type of field 'BeatSaberHTTPStatus.Plugin:levelSceneSetupData' (4) due to: Could not resolve type with token 0100002c (from typeref, class/assembly StandardLevelSceneSetupDataSO, Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null) assembly:Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null type:StandardLevelSceneSetupDataSO member:(null) signature:<none>
  │   at System.Reflection.MonoCMethod.InternalInvoke (System.Object obj, System.Object[] parameters) [0x00018] in <f2e6809acb14476a81f399aeb800f8f2>:0 
  │   at System.RuntimeType.CreateInstanceMono (System.Boolean nonPublic) [0x000a8] in <f2e6809acb14476a81f399aeb800f8f2>:0 
  │   at System.RuntimeType.CreateInstanceSlow (System.Boolean publicOnly, System.Boolean skipCheckThis, System.Boolean fillCache, System.Threading.StackCrawlMark& stackMark) [0x00009] in <f2e6809acb14476a81f399aeb800f8f2>:0 
  │   at System.RuntimeType.CreateInstanceDefaultCtor (System.Boolean publicOnly, System.Boolean skipCheckThis, System.Boolean fillCache, System.Threading.StackCrawlMark& stackMark) [0x00027] in <f2e6809acb14476a81f399aeb800f8f2>:0 
  │   at System.Activator.CreateInstance (System.Type type, System.Boolean nonPublic) [0x00020] in <f2e6809acb14476a81f399aeb800f8f2>:0 
  │   at System.Activator.CreateInstance (System.Type type) [0x00000] in <f2e6809acb14476a81f399aeb800f8f2>:0 
  │   at IllusionInjector.PluginManager.LoadPluginsFromFile (System.String file, System.String exeName) [0x00047] in <be160f5ca99440c98e1b1995202ab6fe>:0 
  │ Running on Unity 2018.2.14f1
```

Other mods for 0.13 work just fine, including beat saber utils. I'm reverting to 0.12.2 for now

**Comment by opl-** [comment]
I'm aware of this issue and already working on an update. I'm hoping to release it tomorrow.

**Comment by renssies** [comment]
Great, I just wanted to make you aware. I'm not going to push you like the rest of the community atm, take your time!

**Comment by opl-** [comment]
Closed by 55e7f224
